stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1951,29 +2110,6 @@ internal constructor(
         return api.removePollVote(messageId, pollId, vote.id)
```

Comment:
What do you think about creating:

```kotlin
removePollVote(
    messageId: String,
    pollId: String,
    voteId: String,
):
```
and deprecate:
```kotlin
removePollVote(
    messageId: String,
    pollId: String,
    vote: Vote,
):
```
?
So that would be align with the specs https://www.notion.so/stream-wiki/Polls-c53f9695fb4e4d9083e5b6b2c49017fd#79a88ba6da7e4fbe8b7e05ce119ba1bc